sympy/functions/special/tests/test_bessel.py

```diff
@@ -142,6 +142,10 @@ def test_besseli_series():
         x**(S(7)/2)/144 + x**(S(9)/2)/2880 + x**(S(11)/2)/86400 + O(x**6)
     assert besseli(-2, sin(x)).series(x, n=4) == besseli(2, sin(x)).series(x, n=4)
 
+    #test for aseries
+    assert besseli(0,x).series(x, oo, n=4) == sqrt(2)*(sqrt(1/x) - (1/x)**(S(3)/2)/8 - \
+        3*(1/x)**(S(5)/2)/128 - 15*(1/x)**(S(7)/2)/1024 + O((1/x)**(S(9)/2), (x, oo)))*exp(x)/(2*sqrt(pi))
```

Comment:
Do we have working results for `-oo` and cases like `I*oo` ?

Reply:
Yes, according to abramowitz and stegun and wolfram alpha we have the above expansion true for negative infinity too, I have added that in the latest commit. Though for I*oo cases I could not find any specific references anywhere.